test/e2e/antreapolicy_test.go

```diff
@@ -3067,6 +3102,7 @@ func TestAntreaPolicy(t *testing.T) {
 		t.Run("Case=ACNPFQDNPolicy", func(t *testing.T) { testFQDNPolicy(t) })
 		t.Run("Case=FQDNPolicyInCluster", func(t *testing.T) { testFQDNPolicyInClusterService(t) })
 		t.Run("Case=ACNPToServices", func(t *testing.T) { testToServices(t) })
+		t.Run("Case=ACNPGroupStandAlonePeer", func(t *testing.T) { testGroupStandAlonePeer(t) })
```

Comment:
validation related e2e tests should go in test group `TestGroupValidateAntreaNativePolicies`. See L3018

Reply:
Thanks. Changed.